### PR TITLE
[WIP]: Storefront eds design

### PR DESCRIPTION
**Pull request by wenche**
solve #125

### CODE REVIEW

**Comment by vnys** [comment]
😛

File: apps/storefront/src/templates/page.jsx [hunk 1]
```diff
@@ -1,85 +1,198 @@
...
   const { slug } = page.fields
   const { currentPage } = page.fields
-  const { currentCategory } = page.fields
   const linkSlug = slug.substr(0, slug.lastIndexOf(currentPage))
-
+  const withTabs = tabs !== null
   const isPublished = (mode || '').toLowerCase() === 'publish'
-
+  const isContentPage = type !== 'landingPage'
+  console.log('stage', process.env.GATSBY_STAGE)
```
Comment (mimarz) [inline apps/storefront/src/templates/page.jsx:95]: Remove comment?

**Pull request merged**